src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -1,6 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Authors>Confluent Inc.;Andreas Heider</Authors>
+    <Description>Confluent's .NET Client for Apache Kafka</Description>
+    <Copyright>Copyright 2016-2017 Confluent Inc., Andreas Heider</Copyright>
+    <PackageTags>Kafka;Confluent;librdkafka</PackageTags>
```

Comment:
Note that for Confluent.Kafka.StrongName, we can modify the description when invoking dotnet build/pack (`/p:Description="Confluent.Kafka strongly signed. Use Confluent.Kafka for normal usage"`)